src/test/java/org/broadinstitute/consent/http/db/UserPropertyDAOTest.java

```diff
@@ -0,0 +1,63 @@
+package org.broadinstitute.consent.http.db;
+
+import org.broadinstitute.consent.http.enumeration.UserFields;
+import org.broadinstitute.consent.http.enumeration.UserRoles;
+import org.broadinstitute.consent.http.models.User;
+import org.broadinstitute.consent.http.models.UserProperty;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.List;
+
+public class UserPropertyDAOTest extends DAOTestHelper {
```

Comment:
I'm very surprised we have not had tests for this DAO class for so long - I missed this completely on the initial migration from mysql -> postgresql